docker/build_conda_packages.sh

```diff
@@ -85,6 +90,7 @@ echo "Running conda build"
 
 # Run with an output folder that is mounted and skip existing to avoid repeated builds
 DOCKER_EXTRA_ARGS="${DOCKER_EXTRA_ARGS[@]}" ${SCRIPT_DIR}/run_container_dev.sh bash -c "${BUILD_SCRIPT}"
+DOCKER_EXTRA_ARGS="${DOCKER_EXTRA_ARGS[@]}" ${SCRIPT_DIR}/run_container_dev.sh bash -c "chown -R ${CUR_UID}:${CUR_GID} .cache"
```

Comment:
Can these be combined into a single command?

Reply:
done